### PR TITLE
Troubleshooting: How to verify membership

### PR DESCRIPTION
**Pull request by prayerslayer**
Just because you can see yourself it doesn't mean others can see you :eyes:

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ Then you need to make sure that
 * This organization exists in the GitHub instance (github.com or your GHE)
 * Everybody who is supposed to approve is a **public** member of this organization
 
+You can change and verify the membership visibility at `<GITHUB INSTANCE/orgs/<YOUR ORG>/people` (e.g. [github.com/orgs/zalando/people](https://github.com/orgs/zalando/people)).
```
Comment (sebastianpoeplau) [inline docs/troubleshooting.md]: I believe you're missing a `>` after `GITHUB INSTANCE` 😬

Comment (prayerslayer) [inline docs/troubleshooting.md]: crap.

Comment (prayerslayer) [inline docs/troubleshooting.md]: thanks for spotting :grin:

**Comment by sebastianpoeplau** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**